pytorch_lightning/callbacks/progress.py

```diff
@@ -412,7 +411,9 @@ def on_train_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, data
 
     def on_validation_start(self, trainer, pl_module):
         super().on_validation_start(trainer, pl_module)
-        if not trainer.running_sanity_check:
+        if trainer.sanity_checking:
+            reset(self.val_progress_bar, sum(trainer.num_sanity_val_batches))
```

Comment:
Because after the changes in this PR, `num_sanity_val_batches` is not defined on `on_sanity_check_start` but `on_validation_start`. Having the number of sanity val batches to run be defined when actually running them makes sense to me.